css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
+	user-select: none;
+}
+
+.spectateBanner::before{
+	content: "👁️ ";
+	font-style: normal;
+}
 .spectateBanner{
 	background-color: #90FF90;
 	cursor: pointer;
-	text-align:center;
+	flex: 0 0 35%;
+	width: 99.5%; /* I think this is due to a padding issue or something? */
+	text-align: center;
+	font-style: italic;
+	background-color: lightgreen;
+	border-style: outset;
+	border-color: lightgreen;
+	font-size: 1em;
+	box-shadow: 0 4px 8px 0 rgba(0,0,0,0.1);
 }
 
 .contentContainer{
 	display: flex;
-	flex-direction: row-reverse;
-	margin-top: var(--topbar-height); /*save room for topbar*/
+	height: 100%;
+}
+
+.sidebarNirnroute{
+	margin-left: 0.5em;
 }
 
 .sidebarContent{
-	width: 500px;
 	display: flex;
 	flex-direction: column;
 	height:100%;
+	overflow: hidden; /* Sidebar links should not scroll with rest of page. */
+}
+#sidebarLinks{
+	padding-left: 0.5em;
+	padding-top: 0.25em;
+	padding-bottom: 0.25em;
```

Comment:
if we want to do short hand we could do "padding: 0.25em 0.5em 0"
it would be the same as this, just in one line.

Reply:
This one is also in the realm of harming readability in favor of less code, and my personal preference is to keep things readable for all of us to have an easier time working with the code. I might be the outlier in that debate though, so if everyone else prefers the more condensed code I'd be willing to change stuff like this.

Reply:
added this change